protogram/src/commonMain/kotlin/com/mattprecious/protogram/protogram.kt

```diff
@@ -94,46 +94,22 @@ private fun ByteString.isProbablyUtf8(): Boolean {
   val byteBuffer = Buffer().write(this)
   while (!byteBuffer.exhausted()) {
     val codePoint = byteBuffer.readUtf8CodePoint()
-    if (codePoint.isISOControl() && !codePoint.isWhitespace()) {
+    if (codePoint.isUnlikelyUtf8()) {
       return false
     }
   }
 
   return true
 }
 
-private fun Int.isISOControl(): Boolean {
-  return (this in 0x00..0x1F) or (this in 0x7F..0x9F)
+private fun Int.isUnlikelyUtf8(): Boolean {
```

Comment:
Can we invert this name and call it `isLikelyBinary` or do `isLikelyUtf8` and flip the return values and negate at the call site?